server_addon/unreal/server/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.0"
```

Comment:
If you don't change settings you don't need to bump version of unreal.

Reply:
I'm adding two settings though! Or should I increase the version only if I change existing ones?

Reply:
Sorry missed them..